File: trunk/client/player.py
#!/usr/bin/python
# coding=UTF-8

#Includes
from PySFML import sf


class Player:
	
	def __init__(self,id,name,table):
		self.name=name
		self.table=table
		self.id=id
		self.hand=[] #Cards in the player's hand or on the table if not player 0		
	
	# add_card : add a card in the player's hand or on the table
	def add_card(self,card):
		self.hand.append(card)
		self.table.add_card(self.id)
		

File: trunk/client_gtk/player.cpp
#include "player.h"
#include <iostream>
#include <string>

Player::Player(std::string name_)
{
	name = name_;
	card_list.push_back(6);
	card_list.push_back(2);
	card_list.push_back(45);
	card_list.push_back(17);
	card_list.push_back(23);
	card_list.push_back(38);
	card_list.push_back(10);
	card_list.push_back(31);
}

Player::~Player()
{
}


